client/sites/marketing/tools/style.scss

```diff
@@ -125,7 +133,19 @@
 .tools__feature-list-item-body {
 	align-items: flex-start;
 	display: flex; // grow this, leave buttons on the bottom of the card
+	flex-direction: column;
 	flex: 1 0 auto;
+	margin-bottom: 1rem;
+}
+
+.tools__feature-list-item-header {
+	display: flex;
+	align-items: center;
+	margin-bottom: 16px;
+	> h1 {
+		font-size: $font-body;
+		font-weight: 500;
+	}
```

Comment:
I think we should remove the inner margins of the `h1` element here so that only the 16px bottom margin applies to the container.

```suggestion
	> h1 {
		font-size: $font-body;
		font-weight: 500;
		margin: 0;
	}
```

| Before | After |
|--------|--------|
| ![CleanShot 2024-11-11 at 13 55 12@2x](https://github.com/user-attachments/assets/4b7956ec-ce30-4b90-ab42-f7a9c729bf3e) | ![CleanShot 2024-11-11 at 13 55 31@2x](https://github.com/user-attachments/assets/cd36c6a1-0d8e-4564-a60b-b38dfdcaf222) |

Reply:
Agreed, adding that